docs/partner-apps/bundleb2b/b2b-edition.md

```diff
@@ -0,0 +1,162 @@
+# The B2B Edition of BigCommerce
+### On this page
+- [B3 integration with BigCommerce](#b3-integration-with-bigcommerce)
```

Comment:
Do we have an okay from Fiona on referring to Bundle B2B as B3?

Reply:
That is the term used in the launch deck, but I am confirming with her.

Reply:
@uwikaiddi, ok, "B3" is fine by her